vision/viscapture/viscapture.go

```diff
@@ -0,0 +1,51 @@
+// Package viscapture implements VisCapture interface returned by the CaptureAllFromCamera vision service method
+package viscapture
+
+import (
+	"image"
+
+	"go.viam.com/rdk/vision"
+	"go.viam.com/rdk/vision/classification"
+	"go.viam.com/rdk/vision/objectdetection"
+)
+
+// VisCapture returns a bundled capture of the requested vision objects.
+type VisCapture interface {
+	Image() image.Image
+	Detections() []objectdetection.Detection
+	Classifications() classification.Classifications
+	PointCloudObject() []*vision.Object
+}
+
+// NewVisCapture returns a VisCapture.
+func NewVisCapture(img image.Image,
+	dets []objectdetection.Detection,
+	class classification.Classifications,
+	obj []*vision.Object,
+) VisCapture {
+	return &capture{
+		image:           img,
+		detections:      dets,
+		classifications: class,
+		pcd:             obj,
+	}
+}
```

Comment:
I'm curious why you chose to make an interface for VisCapture, rather than just making the `struct` Public. Was it in order so that the output could not be modified by a user? In that case, what you can do is make the struct public, but the member variables private, and then have the methods on it to retrieve the values from the member variables (as you have already done)